sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -295,15 +298,17 @@ struct _pi_context : _pi_object {
 const pi_uint32 DynamicBatchStartSize = 4;
 
 struct _pi_queue : _pi_object {
-  _pi_queue(ze_command_queue_handle_t Queue, pi_context Context,
+  _pi_queue(ze_command_queue_handle_t Queue,
+            ze_command_queue_handle_t CopyQueue, pi_context Context,
             pi_device Device, pi_uint32 BatchSize)
-      : ZeCommandQueue{Queue}, Context{Context}, Device{Device},
+      : ZeComputeCommandQueue{Queue},
+        ZeCopyCommandQueue{CopyQueue}, Context{Context}, Device{Device},
         QueueBatchSize{BatchSize > 0 ? BatchSize : DynamicBatchStartSize},
         UseDynamicBatching{BatchSize == 0} {}
 
   // Level Zero command queue handle.
-  ze_command_queue_handle_t ZeCommandQueue;
-
+  ze_command_queue_handle_t ZeComputeCommandQueue;
+  ze_command_queue_handle_t ZeCopyCommandQueue;
```

Comment:
describe them separately, and mention that the copy-queue might not exists at all

Reply:
Done

Reply:
Done. Thanks